backstory/src/main/java/com/google/sps/story/StoryManager.java

```diff
@@ -30,4 +30,15 @@
    * @param factory StoryManagerRequestFactory to use for HttpRequests.
    */
   public void setRequestFactory(StoryManagerRequestFactory factory);
+
+  /**
+   * Cycles to next serviceUrl to a backup url.
```

Comment:
Is next serviceUrl the same as backup url? Slightly confused about this comment.